src/modules/new-request-form/NewRequestForm.tsx

```diff
@@ -78,6 +79,12 @@ export function NewRequestForm({
           case "organization_id":
           case "tickettype":
             return <DropDown field={field} />;
+          case "cc_email":
+            return (
+              <Suspense fallback={<></>}>
+                <CcField field={field} />
```

Comment:
At some point we might need some error handling when lazy loading, e.g. providing a regular text field that asks for emails separated by `,`. In that case, it might be better to not lazy load here. No need to address now though.

Reply:
I am not sure if it makes sense to have a functional fallback component because we would need to support 2 different implementations. For example, with the regular text field, we would still need to parse the value and create some hidden inputs to submit the values.

Reply:
I agree it's maybe it's overkill for error handling 👍🏼 
Let's see how large the js payloads will become. I'm thinking that in some extreme use case, e.g. Disney we might need to do some optimization and that may mean offering "smaller" components. Just something to keep in mind for later.